common/device-model.cpp

```diff
@@ -2741,13 +2744,16 @@ namespace rs2
                 {
                     bool stop_recording = false;
 
-                    ImGui::SetCursorPos({ windows_width - 35, pos.y + 3 });
+                    ImGui::SetCursorPos({ windows_width - 42, pos.y + 3 });
```

Comment:
What is the affect here?
Why 42?
don't we want it according to the font size?

Reply:
Please check

Reply:
All switches look good.